db/src/main/scala/tamer/db/DbSetup.scala

```diff
@@ -1,75 +1,92 @@
 package tamer
 package db
 
-import doobie.Query0
-import zio.UIO
-
 import java.time.Instant
-import scala.concurrent.duration.FiniteDuration
 
-sealed abstract trait QueryBuilder[-S, V] {
+import doobie.Query0
+import doobie.implicits._
+import doobie.util.transactor.Transactor
+import fs2.Stream
+import log.effect.zio.ZioLogWriter.log4sFromName
+import zio._
+import zio.duration._
```

Comment:
favouring `zio.duration.Duration` (which is a type alias for `java.time.Duration`) over `scala.concurrent.duration.Duration` and `scala.concurrent.duration.FiniteDuration` - better ergonomics overall